.code-samples.meilisearch.yaml

```diff
@@ -551,6 +559,8 @@ get_pagination_settings_1: |-
 update_pagination_settings_1: |-
   client.index('books').updateSettings({ pagination: { maxTotalHits: 100 }})
 reset_pagination_settings_1: |-
```

Comment:
Should be added here and on line 558 as well

Reply:
ok.